system-variables.md

```diff
@@ -328,16 +328,16 @@ MPP 是 TiFlash 引擎提供的分布式计算框架，允许节点之间的数
 
 - 作用域：SESSION | GLOBAL
 - 默认值：`10240`
-- 范围：`[0, 9223372036854775807]`
-- 单位为行数。如果 join 的对象为子查询，优化器无法估计子查询结果集大小，在这种情况下通过结果集行数判断。如果子查询的行数估计值小于该变量，则选择 Broadcast Hash Join 算法。否则选择 Shuffled Hash Join 算法。
+- 范围：`[-1, 9223372036854775807]`
+- 单位为行数。如果 join 的对象为子查询，优化器无法估计子查询结果集大小，在这种情况下通过结果集行数判断。如果子查询的行数估计值小于该变量，则选择 Broadcast Hash Join 算法。否则选择 Shuffled Hash Join 算法。当值为 -1 时，默认选择 broadcast_join 算法。
```

Comment:
add a ref to https://docs.pingcap.com/zh/tidb/dev/use-tiflash#mpp-%E6%A8%A1%E5%BC%8F%E7%9A%84%E7%AE%97%E6%B3%95%E6%94%AF%E6%8C%81?